### ISSUE TITLE
m2-cert.pem verification fails

### ISSUE THREAD

**Issue by aidenkeating** [zedshaw/mongrel2#247]
This issue only started today, I think the m2-cert.pem is now invalid.

I installed mongrel2 fine however during one of the tests (cert_tests) the following occurs.

```
Running unit tests:
tests/bstr_tests PASS
tests/cache_tests PASS
ERROR in test tests/cert_tests:
----
RUNNING: ./tests/cert_tests
FAILED: failed to verify cert m2-cert.pem
Tests run: 1
make: *** [tests] Error 1
```

I'm assuming it's because the certificate is now expired however I'm not sure.

**Comment by wysman** [comment]
Confirmed, same as one year ago.
Maybe we can create a certificate for 10 years instead of just one.

**Comment by jasom** [comment]
Fixed by #245
